Dockerfile

```diff
@@ -1 +1,16 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/hello-world:latest
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+WORKDIR /app
+RUN useradd --create-home --user-group arkcase \
+        && mkdir /app/tmp \
+        && chown -R arkcase:arkcase /app 
+        
+    
+USER arkcase
+COPY --chown=arkcase ./config-server-2021.03-SNAPSHOT.jar /app/config-server.jar 
```

Comment:
where do the files `config-server-2021.03-SNAPSHOT.jar` and `start.sh` come from?  I don't see them in the repository or in the pull request.

Reply:
Maybe in this Dockerfile you could either provide (through an environment variable) or specify in the Dockerfile, what version of the config server you want to build, then you could retrieve that version from Nexus?

Reply:
jar file location will be given at build time with `RESOURCE_PATH` arg. Also added cloudconfig version as an argument. So when building both jar location and cloudconfig version will given at build time.